lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -672,6 +704,9 @@ class MailboxDashBoardController extends ReloadableController {
       destinationMailboxId = success.destinationMailboxId;
       moveAction = success.moveAction;
       emailActionType = success.emailActionType;
+      if (Get.isRegistered<ThreadController>()){
+        Get.find<ThreadController>().refreshEmailChanges(currentEmailState: success.currentEmailState);
+      }
```

Comment:
Why dashboard need to care to the threadView? It quite not clean? Please raise the viewState in dashboard and ThreadView should listen to it